nml/global_constants.py

```diff
@@ -1224,6 +1224,7 @@ def config_flag(name, info, pos):
     "dynamic_engines"         : 0x78,
     "variable_runningcosts"   : 0x7E,
     "256_persistent_registers": 0x80,
```

Comment:
Commit message again (this time https://github.com/OpenTTD/OpenTTD/commit/1478fa93b344fc25000725dd143109e4def24f0b)